### ISSUE TITLE
SD2VITA mounted as uma0 instead (alternate version)

### ISSUE THREAD

**Issue by itsminht** [xyzz/gamecard-microsd#27]
Is it possible for each update to this plugin, you also release an alternate version in which the sd2vita card is mounted as uma0 instead of ux0? This would be extremely helpful for the few that use this plugin the other way around.

**Comment by Avrution** [comment]
Seconded.  I'd love to have the wakeup fix working, but need to mount as uma0.

**Comment by xPandamon** [comment]
Yep, i'm up for that. The sporadic wakeups on storagemgr are super annoying. This one: https://github.com/Ninoh-FOX/gamecard-microsd-uma0/releases doesn't seem to be working properly.
